### PR TITLE
make use of Unix-domain socket + yum depreciated

### PR DESCRIPTION
**Pull request by gabx**
When database and Mattermost are on the same machine, best to communicate is to use the unix socket mechanism. It offers tighter security and a faster connection.
- Yum is no longer used in Red Hat family. `dnf` is the new package manager.

### CODE REVIEW

**Comment by esethna** [comment]
Thanks @gabx, Does this PR replace #567? It seems that the PR's have conflicts with each other.

**Comment by gabx** [comment]
@esethna well, it will be easier to remove the #567 and keep this one indeed. It is more complete and detailed.

**Comment by esethna** [comment]
Thanks, closed the old one. Queuing this one for review.

**Comment by gabx** [comment]
In case you want some explanations about the reasons for my PR, you will find me on your mattermost team, user tth. Do not hesitate to ping when needed.

File: source/install/ee-prod-rhel-7.rst [hunk 1]
```diff
@@ -156,24 +163,20 @@ Set up Mattermost Server
    -  Stop the server for now by typing ``Ctrl-C``
 
 8. Set up Mattermost to use the systemd init daemon which handles
-   supervision of the Mattermost process:
-
-   -  ``sudo touch /etc/systemd/system/mattermost.service``
-   -  ``sudo vi /etc/systemd/system/mattermost.service``
-   -  Copy the following lines into
-      ``/etc/systemd/system/mattermost.service``
+   supervision of the Mattermost process. Create and edit ``/etc/systemd/system/mattermost.service``
```
Comment (lfbrock) [inline source/install/ee-prod-rhel-7.rst]: Should we move "Create and edit" to a new line?

**Comment by lfbrock** [comment]
+1 other than the comment above

**Comment by esethna** [comment]
Hi @gabx, I submitted a small PR on your repo for a few formatting tweaks https://github.com/gabx/docs/pull/1

Other than these it looks good, please review when you have a chance?

**Comment by gabx** [comment]
On Mon, Oct 24, 2016 at 7:48 PM Eric Sethna <notifications@github.com>
wrote:

> Hi @gabx <https://github.com/gabx>, I submitted a small PR on your repo
> for a few formatting tweaks gabx#1 <https://github.com/gabx/docs/pull/1>
>
> Other than these it looks good, please review when you have a chance?
>
It has be done and merged. Please tell if anything is missing.

**Comment by esethna** [comment]
Thanks again for the contribution! Merging.

**Pull request merged**